From 0059f0f2ef2b2c807d354afbfaf95ac4023de232 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Thu, 26 Mar 2020 15:38:54 +0800
Subject: [PATCH] ARM: dts: rockchip: rv1126 add pinctrl for emmc but not for
 evb

Since rv1126 evb has both spi nor flash and emmc on board, the
emmc driver not want to set pinctrl, so delete pinctrl property
of emmc node.

Change-Id: I85e9bdfa4ef7d09f306dc43586cbf57feb1edfbb
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm/boot/dts/rv1126.dtsi         | 2 ++
 arch/arm/boot/dts/rv11xx-evb-v10.dtsi | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/rv1126.dtsi b/arch/arm/boot/dts/rv1126.dtsi
index 7c11a6d6a072..50a52399deb2 100644
--- a/arch/arm/boot/dts/rv1126.dtsi
+++ b/arch/arm/boot/dts/rv1126.dtsi
@@ -1435,6 +1435,8 @@
 		clock-names = "biu", "ciu", "ciu-drive", "ciu-sample";
 		fifo-depth = <0x100>;
 		max-frequency = <200000000>;
+		pinctrl-names = "default";
+		pinctrl-0 = <&emmc_clk &emmc_cmd &emmc_bus8>;
 		power-domains = <&power RV1126_PD_NVM>;
 		status = "disabled";
 	};
diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
index d51a04d75fca..83120aaa5bed 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
@@ -420,6 +420,8 @@
 	mmc-hs200-1_8v;
 	rockchip,default-sample-phase = <90>;
 	supports-emmc;
+	/delete-property/ pinctrl-names;
+	/delete-property/ pinctrl-0;
 	status = "okay";
 };
 
-- 
2.35.3

